docs/index.bs

```diff
@@ -3481,93 +3452,79 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
           1. Let |cachedRequest| be the |index|th element in |requestArray|.
           1. If |cachedResponse|'s [=Response/response=]'s [=response/header list=] contains no <a>header</a> [=header/named=] \`<code>Vary</code>\`, or |options|.{{CacheQueryOptions/ignoreVary}} is true, then:
-              1. Add an array [|cachedRequest|, |cachedResponse|] to |resultArray|.
+              1. [=list/Append=] |cachedRequest|/|cachedResponse| to |resultList|.
               1. Continue to the next iteration of the loop.
           1. Let |varyHeaders| be the array containing the elements corresponding to the [=http/field-values=] of the <a>Vary</a> header.
           1. Let |matchFailed| be false.
           1. For each |f| in |varyHeaders|:
               1. If |f| matches "<code>*</code>", or the result of running |cachedRequest|.{{Request/headers}} object's {{Headers/get(name)}} method with |f| as the argument does not match the result of running |request|.{{Request/headers}} object's {{Headers/get(name)}} method with |f| as the argument, then:
                   1. Set |matchFailed| to true.
                   1. Break the loop.
-          1. If |matchFailed| is false, add an array [|cachedRequest|, |cachedResponse|] to |resultArray|.
-      1. Return |resultArray|.
+          1. If |matchFailed| is false, [=list/append=] |cachedRequest|/|cachedResponse| to |resultList|.
+      1. Return |resultList|.
   </section>
 
   <section algorithm>
     <h3 id="batch-cache-operations-algorithm"><dfn>Batch Cache Operations</dfn></h3>
 
       : Input
-      :: |operations|, an array of  {{CacheBatchOperation}} dictionary objects
+      :: |operations|, a [=list=] of {{CacheBatchOperation}} dictionary objects
       : Output
-      :: |promise|, a <a>promise</a> resolves with an array of {{Response}} objects
+      :: |promise|, a [=promise=] that resolves with a [=request response list=]
 
+      1. Let |cache| be the [=context object=]'s [=request response list=].
       1. Let |promise| be a <a>promise</a> resolved with no value.
       1. Return the result of <a>transforming</a> |promise| with a fulfillment handler that performs the following substeps <a>in parallel</a>:
-          1. Let |itemsCopy| be a new <a>request to response map</a> that is a copy of its <a>context object</a>'s <a>request to response map</a>.
-          1. Let |addedRecords| be an empty array.
+          1. Let |itemsCopy| be a new [=request response list=] that is a copy of |cache|.
+          1. Let |addedItems| be an empty [=list=].
           1. Try running the following substeps atomically:
-              1. Let |resultArray| be an empty array.
-              1. For each |operation| in |operations|:
+              1. Let |resultList| be an empty [=list=].
+              1. [=list/For each=] |operation| in |operations|:
                   1. If |operation|.{{CacheBatchOperation/type}} matches neither "delete" nor "put", <a>throw</a> a <code>TypeError</code>.
```

Comment:
We should probably make this an enum or list of possible values.

Reply:
For now, I made it as possible values for the newly defined cache batch operations's type item.